src/Infrastructure/Multitenancy/HideSecureConnectionString.cs

```diff
@@ -0,0 +1,98 @@
+using MySqlConnector;
+using Npgsql;
+using Oracle.ManagedDataAccess.Client;
+using System.Data.SqlClient;
+using DN.WebApi.Infrastructure.Common;
+
+namespace DN.WebApi.Infrastructure.Multitenancy
+{
+    public class HideSecureConnectionString
```

Comment:
I don't really like the name... it's hiding stuff, or it's secure... but not both...

Also don't really like that it's static... Why not make a service out of it... can then be easily replaced with another implementation if someone wants to:

Something like:

```c#
interface IConnectionStringSecurer
{
    string MakeSecure(string connectionString, string dbProvider);
}

```

Granted "Securer" is also not the best of names... but at least it signals its intention more ;-)

Reply:
called IMakeSecureConnectionString.
Actually, you were right. I don't have much imagination with names. :D